acl/src/test/java/org/apache/rocketmq/acl/plain/PlainPermissionManagerTest.java

```diff
@@ -53,10 +54,12 @@ public class PlainPermissionManagerTest {
     PlainAccessConfig plainAccessConfig = new PlainAccessConfig();
     Set<Integer> adminCode = new HashSet<>();
 
-    private static final String PATH = PlainPermissionManagerTest.class.getResource(File.separator).getFile();
+    private static final String PATH = "src/test/resources";
```

Comment:
This does not seem to need to be modified to be compatible with multiple system separators

Reply:
But when I run test case with befor code, not passed on windows platform

Reply:
Regrettably, I don't have Windows system so I can't provide more information. I would like to know if this format `src/test/resources` supports Windows now